pkg/controllers/provisioning/v1alpha1/allocation/controller.go

```diff
@@ -110,5 +117,11 @@ func (c *Controller) Reconcile(ctx context.Context, object controllers.Object) e
 			zap.S().Errorf("Continuing after failing to bind, %s", err.Error())
 		}
 	}
-	return nil
+	return reconcile.Result{}, nil
+}
+
+func (c *Controller) Watches(context.Context) (source.Source, handler.EventHandler, builder.WatchesOption) {
+	return &source.Kind{Type: &v1.Pod{}},
+		&handler.EnqueueRequestForObject{},
+		builder.WithPredicates(predicate.NewPredicateFuncs(func(object client.Object) bool { return false }))
```

Comment:
can you do builder.WithPredicates()?

Reply:
With an empty `builder.WithPredicates()` we would enqueue a `reconcile.Request` for every event on a Pod here. Since we don't have watch properly implemented right now, this essentially says to not enqueue requests at all, where the `GenericController.Reconcile()` function requeues a request every 5 seconds as currently implemented.

Reply:
Sounds good.